analytics/policy/services/business_licence_client.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import urllib3
+from models.address import Address
+from models.listing import Listing
+from policy.utils.dict_utils import merge_dicts
+from urllib3.exceptions import HTTPError
+
+
+class BusinessLicenceClient:
+    BASE_URL = "https://opendata.vancouver.ca/api/explore/v2.1/catalog/datasets/business-licences/records"
+
+    def __init__(self) -> None:
+        self.session = urllib3.PoolManager()
+
+    def filter_by_licence_number(self, licence_number: str) -> dict:
+        # Omit licence_number validity check since it should be checked during policy evaluation
+        return {"where": f'licencenumber="{licence_number}"'}
+
+    def filter_by_short_term_rental_business(self) -> dict:
+        return {"where": 'businesstype="Short-Term Rental"'}
+
+    def licence_status_query(self) -> dict:
+        return {"select": "status"}
+
+    def _make_request(self, fields: dict) -> dict:
+        try:
+            response = self.session.request(
+                method="GET", url=self.BASE_URL, fields=fields
+            )
+            return json.loads(response.data)
+        except HTTPError as e:
```

Comment:
what about general exceptions? are you going to log general exception as well?

Reply:
The request errors have HTTPError as their base class, so this will log all request errors.
I checked the doc here
 https://urllib3.readthedocs.io/en/stable/reference/urllib3.exceptions.html#urllib3.exceptions.HTTPError

Reply:
well you wrapped the whole thing into a try catch, but you are only catching the request errors. not other general errors. errors that occur from json.load will not be captured.

Reply:
I see, will add exception for json.loads